Settings.bundle/com.mono0926.LicensePlist.latest_result.txt

```diff
@@ -2,7 +2,7 @@ name: Crypto, nameSpecified: , owner: soffes, version: 0.6.0
 
 name: GrowingTextView, nameSpecified: , owner: KennethTsang, version: 0.7.2
 
-name: SDWebImage, nameSpecified: , owner: SDWebImage, version: 5.5.2
+name: SDWebImage, nameSpecified: , owner: SDWebImage, version: 5.4.1
```

Comment:
Was this change intended?

Reply:
Not not really. I tried to reset it but license-plist keeps regenerating it when I build the project

Reply:
Maybe it helps to run the update dependency script locally?

Reply:
Or have you tried to use the Xcode Update Swift Package Button? Mh..

Reply:
I ended up with version 5.6 after manually updating

Reply:
I think this is way better than downgrading 😄